crates/prompt_gateway/src/stream_context.rs

```diff
@@ -234,7 +245,7 @@ impl StreamContext {
                             "description embeddings not found for prompt target name: {}",
                             prompt_name
                         );
-                        return (prompt_name.clone(), f64::NAN);
+                        return (prompt_name.clone(), 0.0);
```

Comment:
why return 0.0 instead of nan?

Reply:
0 tells there is no similarity, this only happens when embeddings are not found